CONTRIBUTING.txt

```diff
@@ -330,6 +330,8 @@ the documentation.
 .. code:: sh
 
     pip install sphinx
```

Comment:
Let's replace this with:

```
pip install -r requirements/docs.txt
```

Reply:
@stefanv Thanks for the review. Now it should be consistent when anyone runs ` make SPHINXOPTS= html`. I should have looked for a custom css file in the first place. :)